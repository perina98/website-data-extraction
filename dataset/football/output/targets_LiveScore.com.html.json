[
    {
        "time": "01:30",
        "team1": "Paraguay",
        "score": "? - ?"
    },
    {
        "time": "23:00",
        "team1": "Brazil",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Lincoln City",
        "score": "? - ?"
    },
    {
        "time": "18:45",
        "team1": "Almere City FC",
        "score": "? - ?"
    },
    {
        "time": "21:00",
        "team1": "TOP Oss",
        "score": "? - ?"
    },
    {
        "time": "14:30",
        "team1": "Slovacko",
        "score": "? - ?"
    },
    {
        "time": "12:00",
        "team1": "IFK Gothenburg",
        "score": "? - ?"
    },
    {
        "time": "09:05",
        "team1": "Western Sydney Wanderers FC",
        "score": "? - ?"
    },
    {
        "time": "11:00",
        "team1": "Eastern Suburbs",
        "score": "? - ?"
    },
    {
        "time": "10:30",
        "team1": "Green Gully Cavaliers",
        "score": "? - ?"
    },
    {
        "time": "11:30",
        "team1": "Brisbane Knights",
        "score": "? - ?"
    },
    {
        "time": "11:30",
        "team1": "Mount Gravatt",
        "score": "? - ?"
    },
    {
        "time": "11:30",
        "team1": "Centenary Stormers",
        "score": "? - ?"
    },
    {
        "time": "11:30",
        "team1": "New Farm United",
        "score": "? - ?"
    },
    {
        "time": "10:30",
        "team1": "Souths United",
        "score": "? - ?"
    },
    {
        "time": "07:35",
        "team1": "Sydney FC",
        "score": "? - ?"
    },
    {
        "time": "11:30",
        "team1": "AC Carina",
        "score": "? - ?"
    },
    {
        "time": "19:00",
        "team1": "Wiener SK",
        "score": "? - ?"
    },
    {
        "team2": "Canc.",
        "team1": "SK Rapid Wien II",
        "score": "? - ?"
    },
    {
        "team2": "Canc.",
        "team1": "Mattersburg (A)",
        "score": "? - ?"
    },
    {
        "time": "19:30",
        "team1": "SC Neusiedl am See 1919",
        "score": "? - ?"
    },
    {
        "time": "19:30",
        "team1": "SV Stripfing",
        "score": "? - ?"
    },
    {
        "time": "19:30",
        "team1": "SC Wiener Viktoria",
        "score": "? - ?"
    },
    {
        "time": "19:00",
        "team1": "Deutschlandsberger SC",
        "score": "? - ?"
    },
    {
        "time": "19:00",
        "team1": "Ried (A)",
        "score": "? - ?"
    },
    {
        "time": "01:00",
        "team1": "Central SC",
        "score": "? - ?"
    },
    {
        "time": "19:00",
        "team1": "Slagelse",
        "score": "? - ?"
    },
    {
        "time": "18:30",
        "team1": "Vordingborg",
        "score": "? - ?"
    },
    {
        "time": "15:30",
        "team1": "Renaissance",
        "score": "? - ?"
    },
    {
        "time": "14:00",
        "team1": "Newcastle United U23",
        "score": "? - ?"
    },
    {
        "time": "14:00",
        "team1": "Sheffield United U23",
        "score": "? - ?"
    },
    {
        "time": "14:00",
        "team1": "Queens Park Rangers U23",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Villefranche Beaujolais",
        "score": "? - ?"
    },
    {
        "time": "20:00",
        "team1": "Cholet",
        "score": "? - ?"
    },
    {
        "time": "19:15",
        "team1": "Frankfurt",
        "score": "? - ?"
    },
    {
        "time": "20:30",
        "team1": "College 1975",
        "score": "? - ?"
    },
    {
        "time": "20:30",
        "team1": "St Joseph's",
        "score": "? - ?"
    },
    {
        "team2": "Canc.",
        "team1": "Manchester 62 FC",
        "score": "? - ?"
    },
    {
        "time": "20:30",
        "team1": "Glacis United",
        "score": "? - ?"
    },
    {
        "time": "20:30",
        "team1": "Europa Point",
        "score": "? - ?"
    },
    {
        "time": "20:30",
        "team1": "Lynx",
        "score": "? - ?"
    },
    {
        "time": "20:00",
        "team1": "Valur",
        "score": "? - ?"
    },
    {
        "time": "20:00",
        "team1": "Breidablik",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Longford Town",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Shamrock Rovers",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Finn Harps",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Drogheda United",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Sligo Rovers",
        "score": "? - ?"
    },
    {
        "time": "20:30",
        "team1": "Treaty United",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Cobh Ramblers",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Shelbourne",
        "score": "? - ?"
    },
    {
        "time": "20:45",
        "team1": "Athlone Town",
        "score": "? - ?"
    },
    {
        "time": "21:00",
        "team1": "Cabinteely",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Afula",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Nof HaGalil",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Umm al-Fahm",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Raanana",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Nir Ramat HaSharon",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Jerusalem",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Beitar Tel Aviv Bat Yam",
        "score": "? - ?"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Rishon LeZion",
        "score": "? - ?"
    },
    {
        "time": "02:00",
        "team1": "Cimarrones de Sonora",
        "score": "? - ?"
    },
    {
        "time": "04:05",
        "team1": "Leones Negros",
        "score": "? - ?"
    },
    {
        "time": "23:00",
        "team1": "Atletico de San Luis",
        "score": "? - ?"
    },
    {
        "time": "23:00",
        "team1": "Real Succes",
        "score": "? - ?"
    },
    {
        "time": "23:00",
        "team1": "FC Victoria",
        "score": "? - ?"
    },
    {
        "time": "23:00",
        "team1": "FC Granicerul",
        "score": "? - ?"
    },
    {
        "time": "20:30",
        "team1": "Miedz Legnica",
        "score": "? - ?"
    },
    {
        "time": "16:30",
        "team1": "Metalist 1925",
        "score": "? - ?"
    },
    {
        "time": "16:30",
        "team1": "FC Obolon Kiev",
        "score": "? - ?"
    },
    {
        "time": "16:30",
        "team1": "FC Hirnyk-Sport",
        "score": "? - ?"
    },
    {
        "time": "16:30",
        "team1": "Veres Rivne",
        "score": "? - ?"
    },
    {
        "time": "16:30",
        "team1": "Alians",
        "score": "? - ?"
    },
    {
        "time": "16:30",
        "team1": "Volyn",
        "score": "? - ?"
    },
    {
        "time": "16:30",
        "team1": "Krystal Kherson",
        "score": "? - ?"
    },
    {
        "time": "16:30",
        "team1": "FC Avanhard Kramatorsk",
        "score": "? - ?"
    },
    {
        "time": "11:00",
        "team1": "SC Dnipro-1 U21",
        "score": "? - ?"
    },
    {
        "time": "11:00",
        "team1": "SC Dnipro-1 U19",
        "score": "? - ?"
    },
    {
        "time": "14:40",
        "team1": "AL Oruba",
        "score": "? - ?"
    },
    {
        "time": "14:40",
        "team1": "Dibba",
        "score": "? - ?"
    },
    {
        "time": "14:45",
        "team1": "Al Bataeh",
        "score": "? - ?"
    }
]