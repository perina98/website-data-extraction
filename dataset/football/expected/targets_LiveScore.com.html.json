[
    {
        "time": "01:30",
        "team1": "Chile",
        "undefined": "? - ?",
        "team2": "Paraguay"
    },
    {
        "time": "23:00",
        "team1": "Colombia",
        "undefined": "? - ?",
        "team2": "Brazil"
    },
    {
        "time": "20:45",
        "team1": "Oxford United",
        "undefined": "? - ?",
        "team2": "Lincoln City"
    },
    {
        "time": "18:45",
        "team1": "MVV Maastricht",
        "undefined": "? - ?",
        "team2": "Almere City FC"
    },
    {
        "time": "21:00",
        "team1": "FC Eindhoven",
        "undefined": "? - ?",
        "team2": "TOP Oss"
    },
    {
        "time": "14:30",
        "team1": "Mlada Boleslav",
        "undefined": "? - ?",
        "team2": "Slovacko"
    },
    {
        "time": "12:00",
        "team1": "Vaalerenga",
        "undefined": "? - ?",
        "team2": "IFK Gothenburg"
    },
    {
        "time": "09:05",
        "team1": "Melbourne City FC",
        "undefined": "? - ?",
        "team2": "Western Sydney Wanderers FC"
    },
    {
        "time": "11:00",
        "team1": "Gold Coast United FC",
        "undefined": "? - ?",
        "team2": "Eastern Suburbs"
    },
    {
        "time": "10:30",
        "team1": "Oakleigh Cannons",
        "undefined": "? - ?",
        "team2": "Green Gully Cavaliers"
    },
    {
        "time": "11:30",
        "team1": "The Gap",
        "undefined": "? - ?",
        "team2": "Brisbane Knights"
    },
    {
        "time": "11:30",
        "team1": "University of Queensland",
        "undefined": "? - ?",
        "team2": "Mount Gravatt"
    },
    {
        "time": "11:30",
        "team1": "Western Spirit",
        "undefined": "? - ?",
        "team2": "Centenary Stormers"
    },
    {
        "time": "11:30",
        "team1": "AC Carina",
        "undefined": "? - ?",
        "team2": "New Farm United"
    },
    {
        "time": "10:30",
        "team1": "Ipswich Knights",
        "undefined": "? - ?",
        "team2": "Souths United"
    },
    {
        "time": "07:35",
        "team1": "Canberra United FC",
        "undefined": "? - ?",
        "team2": "Sydney FC"
    },
    {
        "time": "11:30",
        "team1": "Annerley",
        "undefined": "? - ?",
        "team2": "AC Carina"
    },
    {
        "time": "19:00",
        "team1": "ASK-BSC Bruck/Leitha",
        "undefined": "? - ?",
        "team2": "Wiener SK"
    },
    {
        "team1": "SK Rapid Wien II",
        "team2": "SC Neusiedl am See 1919",
        "undefined": "? - ?"
    },
    {
        "team2": "Mattersburg (A)",
        "team1": "ASV Drassburg",
        "undefined": "? - ?"
    },
    {
        "time": "19:30",
        "team1": "ASV Drassburg",
        "undefined": "? - ?",
        "team2": "SC Neusiedl am See 1919"
    },
    {
        "time": "19:30",
        "team1": "FC Marchfeld Mannsdorf",
        "undefined": "? - ?",
        "team2": "SV Stripfing"
    },
    {
        "time": "19:30",
        "team1": "Wiener Neustadt",
        "undefined": "? - ?",
        "team2": "SC Wiener Viktoria"
    },
    {
        "time": "19:00",
        "team1": "Bad Gleichenberg",
        "undefined": "? - ?",
        "team2": "Deutschlandsberger SC"
    },
    {
        "time": "19:00",
        "team1": "SC Weiz",
        "undefined": "? - ?",
        "team2": "Ried (A)"
    },
    {
        "time": "01:00",
        "team1": "Sport Recife",
        "undefined": "? - ?",
        "team2": "Central SC"
    },
    {
        "time": "19:00",
        "team1": "HIK",
        "undefined": "? - ?",
        "team2": "Slagelse"
    },
    {
        "time": "18:30",
        "team1": "Greve",
        "undefined": "? - ?",
        "team2": "Vordingborg"
    },
    {
        "time": "15:30",
        "team1": "TP Mazembe",
        "undefined": "? - ?",
        "team2": "Renaissance"
    },
    {
        "time": "14:00",
        "team1": "Fulham U23",
        "undefined": "? - ?",
        "team2": "Newcastle United U23"
    },
    {
        "time": "14:00",
        "team1": "Sheffield Wednesday U23",
        "undefined": "? - ?",
        "team2": "Sheffield United U23"
    },
    {
        "time": "14:00",
        "team1": "Watford FC U23",
        "undefined": "? - ?",
        "team2": "Queens Park Rangers U23"
    },
    {
        "time": "18:00",
        "team1": "Sete",
        "undefined": "? - ?",
        "team2": "Villefranche Beaujolais"
    },
    {
        "time": "20:00",
        "team1": "FC Bastia-Borgo",
        "undefined": "? - ?",
        "team2": "Cholet"
    },
    {
        "time": "19:15",
        "team1": "Bayer Leverkusen",
        "undefined": "? - ?",
        "team2": "Frankfurt"
    },
    {
        "time": "20:30",
        "team1": "Bruno's Magpies",
        "undefined": "? - ?",
        "team2": "College 1975"
    },
    {
        "time": "20:30",
        "team1": "Europa FC",
        "undefined": "? - ?",
        "team2": "St Joseph's"
    },
    {
        "team1": "Manchester 62 FC",
        "team2": "FC Boca Juniors",
        "undefined": "? - ?"
    },
    {
        "time": "20:30",
        "team1": "Lincoln Red Imps FC",
        "undefined": "? - ?",
        "team2": "Glacis United"
    },
    {
        "time": "20:30",
        "team1": "Lions Gibraltar",
        "undefined": "? - ?",
        "team2": "Europa Point"
    },
    {
        "time": "20:30",
        "team1": "Mons Calpe SC",
        "undefined": "? - ?",
        "team2": "Lynx"
    },
    {
        "time": "20:00",
        "team1": "UMF Selfoss",
        "undefined": "? - ?",
        "team2": "Valur"
    },
    {
        "time": "20:00",
        "team1": "FH Hafnarfjordur",
        "undefined": "? - ?",
        "team2": "Breidablik"
    },
    {
        "time": "20:45",
        "team1": "Bohemian FC",
        "undefined": "? - ?",
        "team2": "Longford Town"
    },
    {
        "time": "20:45",
        "team1": "Derry City",
        "undefined": "? - ?",
        "team2": "Shamrock Rovers"
    },
    {
        "time": "20:45",
        "team1": "Dundalk",
        "undefined": "? - ?",
        "team2": "Finn Harps"
    },
    {
        "time": "20:45",
        "team1": "St. Patrick's Athletic",
        "undefined": "? - ?",
        "team2": "Drogheda United"
    },
    {
        "time": "20:45",
        "team1": "Waterford FC",
        "undefined": "? - ?",
        "team2": "Sligo Rovers"
    },
    {
        "time": "20:30",
        "team1": "Bray Wanderers",
        "undefined": "? - ?",
        "team2": "Treaty United"
    },
    {
        "time": "20:45",
        "team1": "Cork City",
        "undefined": "? - ?",
        "team2": "Cobh Ramblers"
    },
    {
        "time": "20:45",
        "team1": "Galway United FC",
        "undefined": "? - ?",
        "team2": "Shelbourne"
    },
    {
        "time": "20:45",
        "team1": "UCD",
        "undefined": "? - ?",
        "team2": "Athlone Town"
    },
    {
        "time": "21:00",
        "team1": "Wexford FC",
        "undefined": "? - ?",
        "team2": "Cabinteely"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Iksal",
        "undefined": "? - ?",
        "team2": "Hapoel Afula"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Ironi Akko",
        "undefined": "? - ?",
        "team2": "Hapoel Nof HaGalil"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Kfar Shalem",
        "undefined": "? - ?",
        "team2": "Hapoel Umm al-Fahm"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Petah Tikva",
        "undefined": "? - ?",
        "team2": "Hapoel Raanana"
    },
    {
        "time": "18:00",
        "team1": "Hapoel Ramat Gan",
        "undefined": "? - ?",
        "team2": "Hapoel Nir Ramat HaSharon"
    },
    {
        "time": "18:00",
        "team1": "Maccabi Ahi Nazareth",
        "undefined": "? - ?",
        "team2": "Hapoel Jerusalem"
    },
    {
        "time": "18:00",
        "team1": "SC Kfar Kasem",
        "undefined": "? - ?",
        "team2": "Beitar Tel Aviv Bat Yam"
    },
    {
        "time": "18:00",
        "team1": "Sektzia Nes Tziona",
        "undefined": "? - ?",
        "team2": "Hapoel Rishon LeZion"
    },
    {
        "time": "02:00",
        "team1": "Jaibos Tampico Madero",
        "undefined": "? - ?",
        "team2": "Cimarrones de Sonora"
    },
    {
        "time": "04:05",
        "team1": "Atl√©tico Morelia",
        "undefined": "? - ?",
        "team2": "Leones Negros"
    },
    {
        "time": "23:00",
        "team1": "Necaxa",
        "undefined": "? - ?",
        "team2": "Atletico de San Luis"
    },
    {
        "time": "23:00",
        "team1": "FC Cahul 2005",
        "undefined": "? - ?",
        "team2": "Real Succes"
    },
    {
        "time": "23:00",
        "team1": "FC Sireti",
        "undefined": "? - ?",
        "team2": "FC Victoria"
    },
    {
        "time": "23:00",
        "team1": "FC Tighina",
        "undefined": "? - ?",
        "team2": "FC Granicerul"
    },
    {
        "time": "20:30",
        "team1": "Korona Kielce",
        "undefined": "? - ?",
        "team2": "Miedz Legnica"
    },
    {
        "time": "16:30",
        "team1": "Ahrobiznes Volochysk",
        "undefined": "? - ?",
        "team2": "Metalist 1925"
    },
    {
        "time": "16:30",
        "team1": "Chornomorets Odessa",
        "undefined": "? - ?",
        "team2": "FC Obolon Kiev"
    },
    {
        "time": "16:30",
        "team1": "FC Kremin Kremenchuk",
        "undefined": "? - ?",
        "team2": "FC Hirnyk-Sport"
    },
    {
        "time": "16:30",
        "team1": "FC Prykarpattia 1981",
        "undefined": "? - ?",
        "team2": "Veres Rivne"
    },
    {
        "time": "16:30",
        "team1": "MFK Mykolaiv",
        "undefined": "? - ?",
        "team2": "Alians"
    },
    {
        "time": "16:30",
        "team1": "Nyva Ternopil",
        "undefined": "? - ?",
        "team2": "Volyn"
    },
    {
        "time": "16:30",
        "team1": "Polissya Zhytomyr",
        "undefined": "? - ?",
        "team2": "Krystal Kherson"
    },
    {
        "time": "16:30",
        "team1": "VPK-Ahro Shevchenkivka",
        "undefined": "? - ?",
        "team2": "FC Avanhard Kramatorsk"
    },
    {
        "time": "11:00",
        "team1": "Olimpik Donetsk U21",
        "undefined": "? - ?",
        "team2": "SC Dnipro-1 U21"
    },
    {
        "time": "11:00",
        "team1": "FC Olimpik Donetsk  U19",
        "undefined": "? - ?",
        "team2": "SC Dnipro-1 U19"
    },
    {
        "time": "14:40",
        "team1": "AL Tawon SC",
        "undefined": "? - ?",
        "team2": "AL Oruba"
    },
    {
        "time": "14:40",
        "team1": "Masfoot Sports Club",
        "undefined": "? - ?",
        "team2": "Dibba"
    },
    {
        "time": "14:45",
        "team1": "Al-Thaid",
        "undefined": "? - ?",
        "team2": "Al Bataeh"
    }
]